examples/gcoap/gcoap_cli.c

```diff
@@ -26,6 +28,22 @@
 #include "net/gcoap.h"
 #include "od.h"
 #include "fmt.h"
+#ifdef MODULE_SOCK_DTLS
+#include "net/credman.h"
+
+#define SOCK_DTLS_GCOAP_TAG (10)
```

Comment:
Why did you choose 10 for this value? Why not 1 or 5?

Reply:
This can be any `unsigned` values. No reason specifically why 10 is chosen here.